src/Components/ArtsyShippingEstimate.tsx

```diff
@@ -546,11 +546,16 @@ const artaLocation = (
   artwork: ArtsyShippingEstimate_artwork$data,
 ): ArtaLocation | null => {
   const shippingOrigin = artwork.shippingOrigin?.split(", ") ?? []
+  let state = ""
+  if (shippingOrigin.length === 3) {
+    state = shippingOrigin[1]
+  }
   const city = shippingOrigin[0]
```

Comment:
#super_minor and not blocking but maybe for better readability and comprehension of what is happening here I would insert this `state` instruction after city in a form like `const state = shippingOrigin.length === 3 ? shippingOrigin[1] : ""` ? Just feel like it might be easier to read this sequentially...

Reply:
good idea!